app/assets/stylesheets/views/_account-settings.css.scss

```diff
@@ -245,3 +245,14 @@
 .paypal-last-steps-list {
   margin-bottom: lines(1.5);
 }
+
+.paypal-currency-information {
+  padding-left: 30px;
+  position: relative;
+
+  .ss-info {
```

Comment:
This will not work if someone uses font-awesome. Maybe the icons could be wrapped in a `span` or maybe you could add an extra class and use that as a selector?